src/pages/auth/LoginPage.jsx

```diff
@@ -1,24 +1,84 @@
-import React from 'react'
-import { useSignInWithGoogle } from 'react-firebase-hooks/auth'
-import Page from '../../components/page/Page'
-import { auth } from '../../firebase.config'
+import React from "react";
+import { Typography, Box, TextField, Divider } from "@mui/material";
+import { styled, useTheme } from "@mui/material/styles";
+import { useSignInWithGoogle } from "react-firebase-hooks/auth";
+import CustomContainer from "../../components/container/CustomContainer";
+import Page from "../../components/page/Page";
+import { auth } from "../../firebase.config";
+import CustomButton from "../../components/button/CustomButton";
+import { Google } from "@mui/icons-material";
+
+const StyledTextField = styled(TextField)(({ theme }) => ({
+  color: "white",
+  width: "100%",
+  // padding: "5px 5px",
+
+  //center placeholder
+  "& .MuiInputBase-input": {
+    fontSize: "1.3rem",
+    fontFamily: theme.typography.subtitle1.fontFamily,
+    "&::placeholder": {
+      textAlign: "center",
+    },
+  },
+}));
 
 function LoginPage() {
-  const [
-    signInWithGoogle,
-    user,
-    loading,
-    error
-  ] = useSignInWithGoogle(auth)
-  
+  const theme = useTheme();
+  const [signInWithGoogle, user, loading, error] = useSignInWithGoogle(auth);
+
   return (
-    <Page>
-      <div>LoginPage</div>
-      <button onClick={() => signInWithGoogle(
-        ['https://www.googleapis.com/auth/contacts.readonly']
-      )}>Sign In with Google</button>
+    <Page showSideBar={false}>
+      <CustomContainer>
+        <Box sx={{ padding: "50px 100px" }}>
+          <Box sx={{ textAlign: "center", paddingBottom: "3rem"  }}>
```

Comment:
It might be worth dropping the font size on smaller view ports in order to allow for smaller mobile clients to see the page.